### ISSUE TITLE
Content Switcher focus styles need updating for Gray 90/100 themes

### ISSUE THREAD

**Issue by sjbeatle** [carbon-design-system/carbon#6465]
## Detailed description
For the Gray 90/100 themes, the content switcher tabs have to distinguishable focus styling to satisfy a11y [checkpoint 2.4.7 (Focus Visible)](https://www.ibm.com/able/guidelines/ci162/focus_visible.html)

> What version of the Carbon Design System are you using?
Latest

> What did you expect to happen?
Distinguishable focus state for content switcher tabs

> What happened instead?
No distinguishable change in focus state styles

> What WCAG 2.1 checkpoint does the issue violate?
2.4.7

## Steps to reproduce the issue

1. Visit the [content switcher documentation](https://www.carbondesignsystem.com/components/content-switcher/code).
2. Choose either the Gray 90 or Gray 100 theme
3. Tab focus into the content switcher

**Note** There is a white border that appears over inactive content switcher tabs when you mouse click and hold, BUT once the click tab becomes the active tab, there is no visual indication that it has focus because the entire background becomes white.

**Comment by dakahn** [comment]
@carbon-design-system/design :paintbrush: 

This could just be a discrepancy in implementation, but wanted to check with design.

**Comment by aagonzales** [comment]
I'll investigate this today.

**Comment by aagonzales** [comment]
This a was design bug. Here is the update that will fix this problem.
- Add a 1px `$ui-01` border inside the the select + focus tab (like we do with primary button in the light themes). 
- Image key:
   - Top: enabled, 
   - Middle: non-select + focus, 
   - Bottom: select + focus

![image](https://user-images.githubusercontent.com/11670886/88688705-40a35b80-d0bf-11ea-9ceb-87ef75f3924c.png)
![image](https://user-images.githubusercontent.com/11670886/88688947-8829e780-d0bf-11ea-8bd2-399d514b2708.png)
